storage/flush.go

```diff
@@ -53,39 +52,25 @@ type flushManager struct {
 func newFlushManager(database database, scope tally.Scope) databaseFlushManager {
 	opts := database.Options()
 	return &flushManager{
-		database:  database,
-		opts:      opts,
-		nowFn:     opts.ClockOptions().NowFn(),
-		blockSize: opts.RetentionOptions().BlockSize(),
-		pm:        opts.PersistManager(),
-		status:    scope.Gauge("flush"),
+		database: database,
+		opts:     opts,
+		nowFn:    opts.ClockOptions().NowFn(),
+		pm:       opts.PersistManager(),
+		status:   scope.Gauge("flush"),
 	}
 }
 
-func (m *flushManager) NeedsFlush(t time.Time) bool {
+func (m *flushManager) NeedsFlush(start time.Time, end time.Time) bool {
```

Comment:
The method in [types.go](https://github.com/m3db/m3db/blob/c93ecb1dc6806088f616b5ff8cf76960e9acba90/storage/types.go#L207-L209) has a comment for semantics. will rename the variables here.